docs/tutorials/vagrant.rst

```diff
@@ -18,7 +18,7 @@ prerequisites
 
     which can also be used with a pager, such as `less`.
 
-You will need to have `Vagrant`_ and `VirtualBox`_ installed on your machine to use
+You will need to have `Vagrant`_ and `VirtualBox`_ (>=1.8.0) installed on your machine to use
```

Comment:
@yaolingling vagrant 1.8.0 has problems, so it's better >=1.8.1 here  https://github.com/RackHD/RackHD/blob/master/example/README.md 
  1.8.0/1.8.1 is the version of vagrant, the virtualbox version is 4.3.x or 5.x.x